### ISSUE TITLE
Auswahl von Europa Server

### ISSUE THREAD

**Issue by manigl** [nograx/ioBroker.zendure-solarflow#26]
Könnten Sie den Adapter für den Europa Server anpassen?

**Comment by nograx** [comment]
Hallo, 

mein Problem dabei ist das mein eigenes System auf dem globalen Server registriert ist. Um den Adapter anzupassen müsste ich das System auf den Europa Server umstellen. Das ist a) aufwendig und b) kann ich das System in der Zeit nicht über meinen ioBroker steuern. Daher kommt dieser Schritt für mich aktuell nicht in Frage. Tut mir leid. 

Sollte ich irgendwann günstig einen zweiten Hub in die Finger bekommen könnte ich das Thema noch mal angehen.

Gruß,
Peter

**Comment by nograx** [comment]
Ich werde im Laufe des Abends eine Version bereitstellen mit welchem der Europa Server getestet werden kann. Bin hier dann auf Feedback angewiesen.

**Comment by manigl** [comment]
Perfekt, werde Feedbacks übermitteln. Danke

**Comment by nograx** [comment]
Hi, theoretisch könnte der EU server jetzt genutzt werden. Login war bei mir zumindest möglich (wenn ich da keinen Bug eingebaut habe). Kann also gerne mal getestet werden.

**Comment by nograx** [comment]
Das ist interessant, Login scheint ja funktioniert zu haben. Die Fehlermeldung deutet darauf hin das mit der Geräte ID was nicht passt. 

Wäre es möglich das du das Gerät mal für meinen Account freigeben könntest?

**Comment by nograx** [comment]
> Moin, habe mir jetzt extra um eine erste Rückmeldung geben zu können mal ein Accound Gemacht. Die umstellung ging bei mir fast Problem los. Das einzige was Probleme gemacht hat war nicht die Serverauswahl sondern das unter Calculation (ordner in iobroker) keine werte angelegt werden und der dann im Protokoll dieses vermerkt. Diese habe ich dann manuel angelegt. Ansonsten läuft. 👍

Dh der Adapter läuft bei dir über den EU Server?

**Comment by funflo** [comment]
> > Moin, habe mir jetzt extra um eine erste Rückmeldung geben zu können mal ein Accound Gemacht. Die umstellung ging bei mir fast Problem los. Das einzige was Probleme gemacht hat war nicht die Serverauswahl sondern das unter Calculation (ordner in iobroker) keine werte angelegt werden und der dann im Protokoll dieses vermerkt. Diese habe ich dann manuel angelegt. Ansonsten läuft. 👍
> 
> Dh der Adapter läuft bei dir über den EU Server?

Jo, hatte ich heute morgen direkt umgestellt von global auf EU, da ich eh noch am rumbasteln war.
ich hatte hierzu aber den adapter (auch die bereits angelegten werte) gelöscht und den neu eingerichtet.

**Comment by manigl** [comment]
> Das ist interessant, Login scheint ja funktioniert zu haben. Die Fehlermeldung deutet darauf hin das mit der Geräte ID was nicht passt.
> 
> Wäre es möglich das du das Gerät mal für meinen Account freigeben könntest?

Welche Mail Adresse soll ich hinterlegen?

**Comment by nograx** [comment]
> > Das ist interessant, Login scheint ja funktioniert zu haben. Die Fehlermeldung deutet darauf hin das mit der Geräte ID was nicht passt.
> > Wäre es möglich das du das Gerät mal für meinen Account freigeben könntest?
> 
> Welche Mail Adresse soll ich hinterlegen?

Kannst du mich per E-Mail an Peter.frommert@outlook.com kontaktieren? Da kann ich dir dann die für den passenden Account nennen.

**Comment by funflo** [comment]
in der 1.1.23 hatte ich keine probleme in der Aktuellen (1.2.1) hatte ich keine Daten mehr bekommen.
ich bin erstmal wieder zur 1.1.23 zurück gegangen.
hier mal einen auszug aus dem log:

info	[onConnected] Connected with MQTT!
debug	[connectMqttClient] Found 0 SolarFlow devices.
debug	[connectMqttClient] Connecting to MQTT client...
info	[onReady] Found 0 SolarFlow devices.
debug	[getDeviceList] Getting device list from Zendure Rest API!
info	[login] Login to Zendure Rest API successful!
debug	[onReady] Using server eu

**Comment by nograx** [comment]
> in der 1.1.23 hatte ich keine probleme in der Aktuellen (1.2.1) hatte ich keine Daten mehr bekommen. ich bin erstmal wieder zur 1.1.23 zurück gegangen. hier mal einen auszug aus dem log:
> 
> info [onConnected] Connected with MQTT! debug [connectMqttClient] Found 0 SolarFlow devices. debug [connectMqttClient] Connecting to MQTT client... info [onReady] Found 0 SolarFlow devices. debug [getDeviceList] Getting device list from Zendure Rest API! info [login] Login to Zendure Rest API successful! debug [onReady] Using server eu

Hast du zufällig einen Hub 2000?

**Comment by funflo** [comment]
Ne, hub1200 (den „alten“)

**Comment by nograx** [comment]
Wäre es für dich ok mir beide Hubs mal freizugeben? Falls ja kontaktier mich doch bitte unter Peter.frommert@outlook.com, dann gebe ich dir die Daten von meinem dev Account… das Problem von manigl wurde so gelöst.

**Comment by nograx** [comment]
@funflo ich nehme an bei dir funktioniert es jetzt mit Version 1.2.2?

**Comment by funflo** [comment]
Jo, hatte eben das update über github gemacht läuft 👍 
mir wurde das nicht zur installation so vorher angeboten weswegen ich es erst jetzt gemacht hatte.
